theano/tensor/tests/test_basic.py

```diff
@@ -2835,11 +2835,7 @@ def test_cast_from_complex_to_real_raises_error(self):
               correct6=(randint(5, 5).astype('int64'),
                         np.array(-1, dtype='int64'),
                         np.array(1, dtype='int64')),
-              # min > max. messed up behaviour, but
-              # should be same as NumPy's
-              correct7=((5 * rand(5, 5)).astype('float64'),
-                        np.array(1, dtype='float64'),
-                        np.array(-1, dtype='float64')),
+              # min > max case removed as numpy has changed
```

Comment:
A better fix could be to modify the expected lambda on line 2819.
It could be something like:
`lambda x, y z: old_expected_behavior if x > y else np.clip(x, y, z)`.

If it is too complicated for you to make that change, I can still merge this PR as is I think.

Thanks for the recent PRs.

Reply:
Done, and also fixed the documentation that previously claimed it matched numpy.